core/src/main/resources/hudson/model/Computer/builds.properties

```diff
@@ -20,4 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-title=Build History on {0}
\ No newline at end of file
+title=Build History on {0}
+disclaimer=This history is not guaranteed to include all subtasks executed on the node, e.g. Jenkins Pipeline subtasks will not be displayed.
```

Comment:
Are users going to know what “subtasks” are? When you write the script, you are using the `node` step, and I do not think the expression “subtask” is used anywhere in the GUI.